application/src/main/java/org/togetherjava/tjbot/features/basic/SuggestionsUpDownVoter.java

```diff
@@ -55,19 +55,26 @@ public void onMessageReceived(MessageReceivedEvent event) {
     }
 
     private static void createThread(Message message) {
-        String title = message.getContentRaw();
 
-        if (title.length() >= TITLE_MAX_LENGTH) {
-            int lastWordEnd = title.lastIndexOf(' ', TITLE_MAX_LENGTH);
+        String threadTitle;
+        String messageContent = message.getContentRaw();
```

Comment:
References should have an immutable over a mutable state wherever possible and appropriate.

```suggestion
        final String threadTitle;
        final String messageContent = message.getContentRaw();
```